contracts/child/ParentTokenMock.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.2;
```

Comment:
Let's remove this mock. I trying to remove all mocks from the code. It isn't making any sense to have contract mocks.

Reply:
this is an example contract for testing our pre-transfer hooks.

Reply:
this is just an example contract we are using to test beforeTransfer function.